styles.css

```diff
@@ -182,6 +189,185 @@ button:disabled {
   color: #c1c7d0;
 }
 
+.middle-section {
+  display: flex;
+  flex-direction: column;
+}
+
+.card {
+  width: 90%;
+  margin: 1rem auto;
+  display: flex;
+  flex-direction: column;
+  border-radius: 1rem;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  padding-bottom: 1rem;
+}
+
+.project-image img {
+  width: 100%;
+}
+
+ul.project-links {
+  list-style: none;
+  display: flex;
+  width: 80%;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.project-title {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: bolder;
+  font-size: 2.4rem;
+  padding-left: 1rem;
+  margin-bottom: -0.5rem;
+  line-height: 2.4rem;
+}
+
+.career-role,
+.year {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.8rem;
+  line-height: 1rem;
+  color: #7a869a;
+}
+
+.project-description p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 1.5rem;
+  color: #344563;
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.my-skills {
+  display: flex;
+  padding-left: 1rem;
+  text-align: center;
+}
+
+.skill-link {
+  margin-right: 1rem;
+  text-decoration: none;
+  padding: 8px;
+  color: #6070ff;
+  background-color: #e5e5e5;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  letter-spacing: 0.03em;
+  border-radius: 0.5rem;
+}
+
+a.see-project-btn {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 1.1rem;
+  line-height: 1.5rem;
+  color: #396df2;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background: #fff;
+  border: 1px solid #396df2;
+  padding: 1rem;
+  width: 40%;
+  border-radius: 1rem;
+  margin-top: 0.5rem;
+  margin-left: 1rem;
+}
```

Comment:
Please consider adding the functionality of `hover and press` on the button. So when the button is hovered or pressed its style changes as follows. To achieve this you can add the following functionality of code into your CSS file.

.see-project-btn:hover{
    background-color: #6070FF;
    color: white;
}

.see-project-btn:active{
    background-color: #2230D2;
    color: white;
}

![image](https://user-images.githubusercontent.com/94223367/219416172-4958d9ca-1fa4-4068-adbb-ef3657bc7b3f.png)